packages/discovery-react-components/src/utils/onErrorCallback.ts

```diff
@@ -0,0 +1,5 @@
+const onErrorCallback = (error: Error, componentStack: string): void => {
+  console.error('Error caught', error, componentStack);
```

Comment:
im surprised you dont need an eslint rule. also what exactly does this look like?